reference/multigrid/pmis_kernels.cpp

```diff
@@ -71,7 +158,59 @@ void classify(std::shared_ptr<const DefaultExecutor> exec,
               const matrix::SparsityCsr<ValueType, IndexType>* strong_dep,
               int* status)
 {
-    GKO_NOT_IMPLEMENTED;
+    const auto nrows = static_cast<IndexType>(strong_dep->get_size()[0]);
+    const auto s_row_ptrs = strong_dep->get_const_row_ptrs();
+    const auto s_col_idxs = strong_dep->get_const_col_idxs();
+    std::vector<char> higher(nrows, 0);
+
+    auto trans_l = strong_dep->transpose(); 
+    auto transposed_sparsity = gko::as<matrix::SparsityCsr<ValueType, IndexType>>(trans_l.get());
+    const auto tr_row_ptrs = transposed_sparsity->get_const_row_ptrs();
+    const auto tr_col_idxs = transposed_sparsity->get_const_col_idxs();
+
+    for (IndexType i=0; i < nrows; i++){
+        if (status[i] == 0) {
+            char check = 'c';
```

Comment:
check gives one of 'a', 'b', 'c' without the meaning, which is hard to maintain in the future.
At least, you should give the meaning in the comment. 
I think the following code can just rely on status without `check`